python/lsst/validate/drp/plot.py

```diff
@@ -48,6 +48,18 @@
          'iqr': 'green', 'rms': 'red'}
 
 
+def makeFilename(prefix, formatStr, **kwargs):
+    """Return a filename for writing to.
+
+    Return prefix_formatStr.format(kwargs) if prefix is not empty, otherwise
+    just return formatStr.format(kwargs).
+    """
+    if prefix is None or prefix == "":
+        return formatStr.format(**kwargs)
+    else:
+        return "_".join((prefix, formatStr)).format(**kwargs)
```

Comment:
This is seems a bit convoluted... why not just interpolate `"{prefix}_{formatStr}"`?

Reply:
I've reworked it a bit, and I think it's clearer now.